packages/core/plugin-api.d.ts

```diff
@@ -140,6 +140,7 @@ export namespace DatabaseInterface {
     tableCatalog?: string;
     defaultValue?: string;
     isNullable: boolean;
+    isPartitionKey?: boolean;
```

Comment:
Thankfully you choose a better naming for those keys :confetti_ball: 

I'll rename isPk and isFk to make it more readable.